src/utils/debug.h

```diff
@@ -6,7 +6,12 @@
 #include "napa/log.h"
 
 #if defined(DEBUG_NAPA)
-    #define NAPA_DEBUG(section, format, ...) LOG_DEBUG(section, format, ##__VA_ARGS__)
+
+#define NAPA_DEBUG(section, format, ...) LOG_DEBUG(section, format, ##__VA_ARGS__)
+
 #else
-    #define NAPA_DEBUG
+
+// Do nothing without generating any "unreferenced variable" warnings.
```

Comment:
The purpose is not exactly good. 

Should the "unreferenced variable" warning comes out if defined DEBUG_NAPA?
If so, we better let it be here in compile time.

Reply:
This is my bad. please ignore that commit..

I updated the definition to `#define NAPA_DEBUG(section, format, ...) ((void)0)` in new commit. This does not deal with "unreferenced variable" warnings, but moves out unnecessary expression evaluation. This is the real purpose.

Reply:
Ooops, I understand that NAPA_DEBUG should mainly target on undefined things in release build.

Yet, currently it is used more to act LOG_DEBUG in our existing code. Although it could used as this, in well defined LOG code, it should be more preferred to use LOG_DEBUG directly.

I think the ((void) 0) is good. Or like other commonly used: do {} while (false).